src/loaders/secretsManager.js

```diff
@@ -0,0 +1,65 @@
+import AWS from 'aws-sdk';
+
+import { CREDSTASH_PREFIX } from '../constants';
+import { convert } from './convert';
+
+
+export function getClient() {
+    return new AWS.SecretsManager();
+}
+
+function convertAllKeys(obj) {
+    if (typeof obj !== 'object') return convert(obj);
+
+    const newObj = {};
+
+    Object.keys(obj).forEach((name) => {
+        newObj[name] = convertAllKeys(obj[name]);
+    });
+
+    return newObj;
+}
+
+export default async function loadFromSecretsManager(metadata) {
+    const version = process.env[`${CREDSTASH_PREFIX}_CONFIG_VERSION`];
+    const environment = process.env[`${CREDSTASH_PREFIX}_ENVIRONMENT`];
+
+    if (!version) {
+        const message = `Must define environment variable: ${CREDSTASH_PREFIX}_CONFIG_VERSION`;
+
+        if (!metadata.debug) {
```

Comment:
Can you concentrate all the `if(!metadata.debug)` in one place? It's scary being spread all over, one mistake and we end up returning empty dictionaries.